### PR TITLE
Add Build Whonix Templates doc

### PR DESCRIPTION
**Pull request by awokd**
Per @adrelanos in https://github.com/QubesOS/qubes-issues/issues/3441.

### CODE REVIEW

**Comment by awokd** [comment]
Debating adding a link in here to https://github.com/QubesOS/qubes-builder/blob/master/doc/TemplateDispVMBuild.md, but does it work on R4.0?

By the way @adrelanos , there is a (new) description of the manual process now in https://www.qubes-os.org/doc/qubes-r3-building/#build-using-manual-steps.

**Comment by andrewdavidwong** [comment]
@adrelanos, would you be willing to review this PR?

**Comment by adrelanos** [comment]
Let's see. I didn't have much luck using `./setup` earlier. Using https://www.whonix.org/wiki/Dev/Qubes#Build_Qubes-Whonix_Templates_-_New_New now which Marek provided in https://github.com/QubesOS/qubes-issues/issues/3441#issuecomment-359447019.

Did you manage to build Qubes-Whonix using these exact steps using a new VM? @awokd

**Comment by awokd** [comment]
I did (once I git pulled `qubes-builder` to the current master).

**Comment by awokd** [comment]
Thanks for looking it over though, @adrelanos . I can see from the release notes for the Whonix 14 builds you have your hands pretty full already!

**Comment by andrewdavidwong** [comment]
Thanks for the update, @adrelanos.

@marmarek, do you want to review this, or should we just merge?

File: building/building-whonix-template.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Continue the build process with:
+
+~~~
+make install-deps
+make get-sources
+~~~
+
+You will often need to edit/update `qubes-src/template-whonix/builder.conf` at this stage to specify the currently shipping Tor Browser version.
+Open it in your favorite editor, then look for "Extra Whonix Build Options" and add/edit the `WHONIX_TBB_VERSION` variable to specify the current version.
```
Comment (marmarek) [inline building/building-whonix-template.md]: It is also possible to set `WHONIX_TBB_VERSION` from main `builder.conf` file (or `override.conf`, if you use one).
Not sure if worth mentioning, but generally I consider more convenient to modify just one config file, not different parts in different places.

Comment (awokd) [reply building/building-whonix-template.md]: Good point; I'm planning on referencing back to this note from the main ISO build doc once this one is "live" so will update it then too.

**Pull request merged**